src/app/components/Article/index.jsx

```diff
@@ -35,18 +49,18 @@ class Article extends Component {
   }
 
   render() {
-    const { headline } = this.state;
+    const { data } = this.state;
+    const { text } = data.blocks[0].model.blocks[0].model.blocks[0].model;
+
     return (
       <Fragment>
         <Helmet htmlAttributes={{ lang: 'en-GB' }}>
           <title>
-            {headline}
+            {text}
           </title>
         </Helmet>
         <Header />
-        <Headline>
-          {headline}
-        </Headline>
+        <MainContent {...this.state} />
```

Comment:
`{...model}` should be passed and destructured here, not `{...this.state}`. This is possible by the change in data structure in the comment [here](https://github.com/bbc/simorgh/pull/247/files#diff-5f296fd8afe33a8853c3f2655172c257R10).